src/Assetic/Filter/ScssphpFilter.php

```diff
@@ -18,23 +18,12 @@
  */
 class ScssphpFilter extends BaseFilter implements DependencyExtractorInterface
 {
-    private $compass = false;
     private $importPaths = [];
     private $customFunctions = [];
     private $formatter;
     private $outputStyle;
     private $variables = [];
 
-    public function enableCompass($enable = true)
-    {
-        $this->compass = (Boolean) $enable;
-    }
-
-    public function isCompassEnabled()
```

Comment:
you should keep these 2 methods (returning `false` here and being a no-op in the other one) but deprecate them. Otherwise, that's a BC break.

Reply:
That's a fair point, although there is a presumption that these methods shouldn't be here anyway given that our 2.0 changelog [explicitly said](https://github.com/assetic-php/assetic/blob/master/CHANGELOG-2.0.md#filters-removed) that Compass was dropped.

Nevertheless, to stay semantic, I'll add them back in and deprecate them.

Reply:
Well, 1.x had a _separate_ CompassFilter based on the ruby compass. This is the one dropped in 2.0

Reply:
@stof You are correct - my apologies.